app/components/primer/details_component.rb

```diff
@@ -12,9 +12,42 @@ class DetailsComponent < Primer::Component
       :dark => "details-overlay details-overlay-dark"
     }.freeze
 
-    with_slot :body, class_name: "Body"
-    with_slot :summary, class_name: "Summary"
+    renders_one :body, class_name: "Body"
+    renders_one :summary, class_name: "Summary"
```

Comment:
```suggestion
    renders_one :body, "Body"
    renders_one :summary, "Summary"
```

Reply:
I'm still seeing the same problem. I'm gunna convert to the lamba format. Still not sure what's going wrong though...